packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -1038,6 +1042,26 @@ describe('TokenDetectionController', () => {
                 },
               },
             });
+            mockNetworkState({
+              networkConfigurationsByChainId: {
+                '0x1': {
+                  name: 'ethereum',
```

Comment:
Since now we expect the code to detect cross networks; does it make sense to update this mock to have more than one network? and that addDetected tokens is working for more than 1 chain?

Reply:
done [a865de4](https://github.com/MetaMask/core/pull/4957/commits/a865de4c701bb58f181d065b8d6234183d844ece)